amivapi/utils.py

```diff
@@ -16,6 +18,20 @@
 from flask import g
 
 
+def token_urlsafe():
+    """Cryptographically random generate a token that can be passed in a URL.
+    The token is created from 256 random bits.
+
+    This function is available as secrets.token_urlsafe in python3.6. We can
+    remove this function when we drop python3.5 support.
+
+    Returns:
+        str: A random string containing only urlsafe characters.
+    """
+    return b64encode(urandom(32)).decode("utf-8").replace("+", "-").replace(
```

Comment:
why does the implementation differ from the py36 `token_urlsafe`?
https://github.com/python/cpython/blob/3.6/Lib/secrets.py

Reply:
Because our code has been there before we knew about that function. But I don't think it makes a difference.